### ISSUE TITLE
Partner with external accessibility team

### ISSUE THREAD

**Issue by jeff-zucker** [solid/deit#5]
We should have resources for designers on accessibility issues and document the ways we support accessibility (e.g. sp.org just decided to drop underlining of links to support neuro-atypical users and to add slight bolding and underline on hover to support low-vision users). 

Maybe we should partner with an external accessibility team (does W3C have one?)  who could provide some guideline docs and a resource for those with questions.

**Comment by csarven** [comment]
I've captured some key initiatives and works at https://solidproject.org/TR/protocol#accessibility-considerations .
